### DISCUSSION TITLE
Improvement of app sidebar design

### DISCUSSION THREAD

**Discussion by DonKoko** [Shelf-nu/shelf.nu#974] [Ideas]
So I was looking at our app's UI and I also had stripe open at the same time and I realized that their sidebar looks so much cleaner. Also it didn't look too overwhelming and that its taking too much space. Here are 2 screenshots for comparison. Both screenshots are taken on the same browser and at 100% scale.

**Stripe**: 
<img width="1800" alt="Screenshot 2024-05-15 at 13 08 52" src="https://github.com/Shelf-nu/shelf.nu/assets/7840007/a9acd24a-8972-41bc-ac35-034ae9fcd8b2">

**Shelf**:
 <img width="1800" alt="Screenshot 2024-05-15 at 13 08 47" src="https://github.com/Shelf-nu/shelf.nu/assets/7840007/4c07f8a2-a512-42d6-90bb-b78be987f391">

So now that we are getting more and more items in the sidebar I feel it starts to get cluttered and a bit overwhelming. 
@jurrejansen @carlosvirreira what do you guys think? Can we rethink/resize a bit the items to make it a bit cleaner?

A few more ideas:
- I really like their way of pinning items to the sidebar. That is complete user agency and we talk about that a lot so it can be nice for the users, especially since we are starting to get more and more items there
- Foldable items - I think this can be very useful for menu items that are related to the same concept. For example we could have:

```
- Bookings
  - Overview
  - Calendar
```
<img width="286" alt="Screenshot 2024-05-15 at 13 13 33" src="https://github.com/Shelf-nu/shelf.nu/assets/7840007/999fa47a-b2d7-4ba5-b04b-bdb034d6fbac">

**Comment by jurrejansen** [comment]
@DonKoko @carlosvirreira all for improving the sidebar and making it more minimal and taking up less space perhaps.

**Amount of menu items**
We could also consider moving some items under Settings (Categories, Tags, Locations)

**Foldable items**
Not sure how often this is going to be relevant. Maybe good to look at planned features/concepts and see if they can be grouped. Also for booking and calendar I feel this could also be an option.

![Bookings _ Master](https://github.com/Shelf-nu/shelf.nu/assets/76112272/f06b02ce-c78d-4eff-934b-ed0658332f4d)
![Calendar](https://github.com/Shelf-nu/shelf.nu/assets/76112272/bb07c7c8-0514-4a26-b782-44f17da0f6f1)
